force-app/main/default/classes/SOQL.cls

```diff
@@ -108,7 +135,7 @@ public with sharing class SOQL {
      * @return A list of Opportunity records.
      */
     public static List<Opportunity> getOpportunitiesByNameKeyword(String keyword) {
-        return null; // Replace null with the variable you used to store the result
+        return Database.query('SELECT Id, Name, Description, Amount from Opportunity where Name like \'%' +  keyword + '%\''); // Replace null with the variable you used to store the result
```

Comment:
This is totally fine, but you could use an intermediate variable here to define your bind variable `keyword` as `keyword = '%' + keyword + '%';`

Depends on your preference for readability though.